### ISSUE TITLE
Issues with loading model BPNet-OSKN

### ISSUE THREAD

**Issue by IshikaLuthra** [kipoi/kipoi#602]
Hi Again,

I am also trying to use the python version of Kipoi and am able to predict and use Basset. But when I try `model = kipoi.get_model('BPNet-OSKN')` I get the following error:

`Using downloaded and verified file: /home/iluthra/.kipoi/models/BPNet-OSKN/downloaded/model_files/model_file/bbe883baef261877bfad07d05feb627d
Traceback (most recent call last):
  File "/home/iluthra/kipoi/lib/python3.6/site-packages/kipoi_utils/utils.py", line 139, in load_obj
    module = imp.load_module(module_name, fp, pathname, description)
  File "/cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.6.10/lib/python3.6/imp.py", line 235, in load_module
    return load_source(name, filename, file)
  File "/cvmfs/soft.computecanada.ca/easybuild/software/2020/avx2/Core/python/3.6.10/lib/python3.6/imp.py", line 172, in load_source
    module = _load(spec)
  File "<frozen importlib._bootstrap>", line 684, in _load
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/iluthra/.kipoi/models/BPNet-OSKN/model.py", line 1, in <module>
    from bpnet.seqmodel import SeqModel
ModuleNotFoundError: No module named 'bpnet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/iluthra/kipoi/lib/python3.6/site-packages/kipoi/model.py", line 183, in get_model
    Mod = load_obj(md.defined_as)
  File "/home/iluthra/kipoi/lib/python3.6/site-packages/kipoi_utils/utils.py", line 144, in load_obj
    raise ImportError("object {} couldn't be imported. Error {}".format(obj_import, str(e)))
ImportError: object model.BPNetOldSeqModel couldn't be imported. Error No module named 'bpnet'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/iluthra/kipoi/lib/python3.6/site-packages/kipoi/model.py", line 191, in get_model
    raise ImportError("Unable to import {}".format(md.defined_as))
ImportError: Unable to import model.BPNetOldSeqModel`

Once again, thank you for this awesome package and all your help :)

**Comment by haimasree** [comment]
Hi Ishika, Thanks for your feedback. Are you running your code inside the conda environment required for BPNet-OSKN?
You can create and activate the environemnt with the following command
```
kipoi env create BPNet-OSKN
source activate kipoi-BPNet-OSKN
```

**Comment by IshikaLuthra** [comment]
Hi, I am not able to use Anaconda, so I was surprised that Basset worked without creating a conda environment. I guess to use BPNet properly I would have to follow the commands you listed above. I think it would be best if I can get the singularity version running instead of python in that case.

Thank you for all your help!

**Comment by haimasree** [comment]
Hi Ishika, Basset needs these packages to run - https://github.com/kipoi/models/blob/master/Basset/model.yaml#L16-L24. Perhaps you already installed them in your python distribution? Do you use something else like pip or poetry for working with virtual environments? If no, then working out of your base python distribution is not a good idea.

**Comment by haimasree** [comment]
Solved in #600
